river/InputConfig.zig

```diff
@@ -265,6 +302,14 @@ pub fn parse(self: *Self, setting: []const u8, value: []const u8) !void {
                 const ret = c.libevdev_event_code_from_name(c.EV_KEY, value.ptr);
                 if (ret < 1) return error.InvalidButton;
                 self.@"scroll-button" = ScrollButton{ .button = @intCast(ret) };
+            } else if (comptime mem.eql(u8, field.name, "map-to-output")) {
+                if (self.@"map-to-output") |mto|
+                    if (mto.output_name) |name|
+                        util.gpa.free(name);
+
+                self.@"map-to-output" = MapToOutput{
+                    .output_name = if (std.mem.eql(u8, value, "disable")) null else try util.gpa.dupe(u8, value),
```

Comment:
This doesn't match the man page docs and is inconsistent with other input configuration settings. 
```suggestion
                    .output_name = if (std.mem.eql(u8, value, "disabled")) null else try util.gpa.dupe(u8, value),
```